static/js/components/LearningResourceCard.js

```diff
@@ -55,11 +59,45 @@ const getPlatform = (object: LearningResourceSummary): string => {
   }
 }
 
+const getPlatformName = object => platformReadableNames[getPlatform(object)]
+
+const getClassName = searchResultUI =>
+  `learning-resource-card ${
+    searchResultUI === SEARCH_LIST_UI ? "list-view" : ""
+  }`.trim()
+
+const formatTopics = (topics: Array<{ name: string }>) =>
+  topics.map(topic => topic.name).join(" ")
```

Comment:
I think this either needs to be a double-space or a comma-space, because it's hard to see where topics are divided. Personal opinion is that commas are a much easier read, although that isn't in the mocks.

Reply:
@pdpinch what would you prefer here? for context, this is the divider to put in-between teh different topics for a course or program when we display them

Reply:
I'll go with double spaces for now

Reply:
(I think commas are easier too, but not in the mocks - easy enough to change later)

Reply:
Sorry I was late for this. The reason for the spacing is to make sure there enough space between clickable (and tappable) areas. I think @abdulkdawson had some reasoning behind not having a comma separator, but I don't recall it right now.